testsuite/documentation/static-run.md

```diff
@@ -28,6 +29,7 @@ To run all standard tests, from the controller:
 export SERVER="${PREFIX}suma3pg.tf.local"
 export CLIENT="${PREFIX}clisles12sp3.tf.local"
 export MINION="${PREFIX}minsles12sp3.tf.local"
+export BUILD_HOST="${PREFIX}bldhsles12sp3.tf.local"
```

Comment:
bldh? Sounds like my dog tapping the keyboard :dog: 
IMHO I don't is that bad to add few more characters to a domain name. build-host-sles12sp3.tf.local
BTW, we do on a SP3 then?

Reply:
Thanks for suggesting an alternative; I am going to switch to build-host-sles12sp3.tf.local.
We are using SP4; I am going to fix al so that.